core/src/main/java/org/arquillian/smart/testing/configuration/ConfigurationItem.java

```diff
@@ -5,27 +5,32 @@
     private final String paramName;
     private String systemProperty;
     private Object defaultValue;
+    private final boolean regExp;
 
     /**
      * Constructor
      * @param paramName needs to match with name of setter method by excluding set/add.
      * @param systemProperty required to overwrite field value.
+     * @param isRegExp true if system property is a regular expression of form my.proeprty.*
      * @param defaultValue to use for the field.
      */
-    public ConfigurationItem(String paramName, String systemProperty, Object defaultValue) {
+    public ConfigurationItem(String paramName, String systemProperty, boolean isRegExp, Object defaultValue) {
         this.paramName = paramName;
         this.systemProperty = systemProperty;
         this.defaultValue = defaultValue;
+        this.regExp = isRegExp;
```

Comment:
Can't we define method which will check for given system property is a refular expression of required form i.e. `my.property.*`? So we'll get rid out of this `isRegExp` params in constructor which is not necessary.

Reply:
so you mean if it ends with `.*` means it is a regular expression right?